From f275dc147bf002835514e3e39bfc41b068b59bc4 Mon Sep 17 00:00:00 2001
Message-ID: <f275dc147bf002835514e3e39bfc41b068b59bc4.1713654571.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1713654571.git.sharlatanus@gmail.com>
References: <cover.1713654571.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 3 Apr 2024 16:26:49 +0100
Subject: [PATCH 20/47] gnu: python-stsci-image: Refresh package style.

* gnu/packages/astronomy.scm (python-stsci-image): Adjust indentation.
[description]: Fix grammar.

Change-Id: I1f4bf751720a3be324644bf417104e36510c62a6
---
 gnu/packages/astronomy.scm | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index b7f7127c24..4fea09067c 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3135,28 +3135,28 @@ (define-public python-stsci-image
   (package
     (name "python-stsci-image")
     (version "2.3.5")
-    (source (origin
-              (method url-fetch)
-              (uri (pypi-uri "stsci.image" version))
-              (sha256
-               (base32
-                "1vnp4256nbdvapa69cmm80sjz11ygxa49abr9nbvssj6nyyp5icb"))))
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "stsci.image" version))
+       (sha256
+        (base32 "1vnp4256nbdvapa69cmm80sjz11ygxa49abr9nbvssj6nyyp5icb"))))
     (build-system pyproject-build-system)
     (arguments
-     (list #:phases #~(modify-phases %standard-phases
-                        (add-before 'check 'build-extensions
-                          (lambda _
-                            ;; Cython extensions have to be built before running
-                            ;; the tests.
-                            (invoke "python" "setup.py" "build_ext"
-                                    "--inplace"))))))
+     (list
+      #:phases
+      #~(modify-phases %standard-phases
+          (add-before 'check 'build-extensions
+            (lambda _
+              ;; Cython extensions have to be built before running the tests.
+              (invoke "python" "setup.py" "build_ext" "--inplace"))))))
     (propagated-inputs (list python-numpy python-scipy))
     (native-inputs (list python-pytest python-setuptools-scm))
     (home-page "https://github.com/spacetelescope/stsci.image")
     (synopsis "Image array manipulation functions")
     (description
-     "This package provides Python modules of @acronym{STScI, Space Telescope
-Science Institute} image array manipulation functions.")
+     "This package provides a Python module to various @acronym{STScI, Space
+Telescope Science Institute} image array manipulation functions.")
     (license license:bsd-3)))
 
 (define-public python-stsci-imagestats
-- 
2.41.0

